cmd/root.go

```diff
@@ -78,49 +67,13 @@ func Execute() error {
 	return NewRootCmd().Execute()
 }
 
-func start() {
-	if flags.larkWebbookURLs == "" {
-		log.Fatal(`The "--lark-urls" or "LARK_URLS" is required`)
+func start(f *flamego.Flame) {
+	if err := hook.Prepare(); err != nil {
+		log.Fatalf("Failed to prepare hook: %v\n", err)
```

Comment:
FYI, `log.Fatalf` (or any `log.XYZf` from stdlib) always prints to a new line, below is effectively the same
```suggestion
		log.Fatalf("Failed to prepare hook: %v", err)
```

Reply:
Done